Practice#5

```diff
@@ -1,5 +1,3 @@
-*Create example of final static collections.
-
 //Assume that we have a function that returns a list of objects
 
 private final static Collection<? super Object> myColl;
```

Comment:
naming conventions?

The initializing of final static Collection is not fine - find out the best solution